### DISCUSSION TITLE
Specification table has no border radius

### DISCUSSION THREAD

**Discussion by rakleed** [mdn/yari#7764] [Ideas]
It would be nice if the table with specifications (`#specifications .standard-table`) had a border radius like other similar elements:
- `.interactive`
- `.section-content pre`
- `.section-content .code-example pre`
- `.sample-code-frame`
- `.bc-table`
- `.metadata`

**Comment by bsmth** [comment]
This is a duplicate of https://github.com/orgs/mdn/discussions/301 so I'm going to close in favor of that one. Thank you :)
